### PR TITLE
Merge 2019 11 29 recovery services

### PR DESCRIPTION
**Pull request by VeryEarly**
## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: tools/Az/Az.psd1 [hunk 1]
```diff
@@ -297,7 +297,6 @@ Az.PrivateDns
 
 Az.RecoveryServices
 * Update references in .psd1 to use relative path
-* Added SoftDelete feature for VM and added tests for softdelete
```
Comment (isra-fel) [inline tools/Az/Az.psd1]: Why is this line removed

Comment (VeryEarly) [reply tools/Az/Az.psd1]: this message was incorrectly been added , already existed in version 2.0.1

**Comment by VeryEarly** [comment]
> Is serialized json generated?

it was generated when I update the release branch last week, so no differences showed in this PR, I did minor version upgrade manually.
